spec/lib/tasks/jobs_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+require "rake"
+
+RSpec.describe "rake db:schedule_jobs" do
+  def queue_adapter_for_test
+    DelayedJobTestAdapter.new
+  end
+
+  before do
+    Rake::Task.define_task(:environment)
+    Rake::Task.define_task(:"db:migrate")
+    Rake::Task.define_task(:"db:schema:load")
+    Rake.application.rake_require "tasks/jobs"
+  end
+
+  it "schedules and enqueues SchoolDataImporterJob" do
```

Comment:
This seems to rely on some state outside the test (namely `SchoolDataImporterJob`), so if we change / add jobs, this test may fail. Can we set up some stubs this so the state remains inside the tests?